ckanext/restricted/i18n/fr/LC_MESSAGES/ckanext-restricted.po

```diff
@@ -41,18 +42,16 @@ msgid "Access Request to resource {0} ({1}) from {2}"
 msgstr "Demande d’accès à la ressource {0} ({1}) par {2}"
 
 #: ckanext/restricted/controller.py:105
-msgid ""
-"Please find below a copy of the access request mail sent. \n"
+msgid "Please find below a copy of the access request mail sent. \n"
 "\n"
 " >> {}"
-msgstr ""
-"Veuillez trouver ci-dessous une copie de l’e-mail de demande d’accès.\n"
+msgstr "Veuillez trouver ci-dessous une copie de l’e-mail de demande d’accès.\n"
 "\n"
 " >> {}"
 
 #: ckanext/restricted/controller.py:131 ckanext/restricted/controller.py:245
 msgid "Bad Captcha. Please try again."
-msgstr "CAPTCHA incorrect. Essayez à nouveau."
+msgstr "CODE incorrect. Essayez à nouveau."
```

Comment:
Not convinced by this translation since CAPTCHA is an acronym/noun that is almost certainly translated the same in [english and french](https://en.bab.la/dictionary/english-french/captcha).   Not sure Ian clocked exactly what it meant.   Think I would not bother Ian about it (as I don't think we even use CAPTCHA at the moment), but I'd just change it to  "CAPTCHA code incorrect. Essayez à nouveau."  Or we just wash our hands of the responsibility and leave it as Ian gave.

Reply:
You're right. I've fixed that and pushed a change.